src/views/Statistics.vue

```diff
@@ -83,12 +94,54 @@ export default {
         },
         { name: 'Areas climbed', value: Object.keys(areas).length },
       ];
-      this.loaded = true;
+    },
+
+    updateItems() {
+      if (!this.userDoc || !this.indexedData || !this.indexedData.routes ||
+          !this.climbDataLoaded) {
+        return;
+      }
+
+      // If the user is on a team, retrieve the user stats from the team
+      // climbs, and also fill in team stats.
+      if (this.teamDoc && this.teamDoc.users) {
+        const users = this.teamDoc.users;
+
+        // Compute the stats for the user.
+        if (auth.currentUser.uid in users) {
```

Comment:
unless you're anti-ternary, how about something like this, which is a lot shorter and feels pretty self-explanatory to me?

```js
const userClimbs = ...;  // from below
const uid = auth.currentUser.uid;
this.itemsUser = this.computeStats(userClimbs[uid] ? [userClimbs[uid]] : []);
this.itemsTeam = this.computeStats(userClimbs);
```

Reply:
I love ternary!